packages/client3/components/design-system/input/input.stories.tsx

```diff
@@ -15,20 +15,19 @@ export const InputStory: ComponentStory<typeof Input> = (args) => (
 );
 
 InputStory.args = {
-  placeholder: "Placeholder",
+  placeholder: "Text Here",
   textSize: "md",
 };
 
 export const DollarInputStory: ComponentStory<typeof DollarInput> = (args) => {
   const { control, setValue, handleSubmit } = useForm<{ amount: string }>();
   return (
     <form onSubmit={handleSubmit((data) => alert(`Amount: ${data.amount}`))}>
-      <div className="mb-4">
-        It is important to keep in mind that the DollarInput component is an
-        adapted controlled input, and as such it can only be used alongside
-        React Hook Form, not by itself. This is considered a best-practice for
-        input elements in this project regardless.
-      </div>
+      <DollarInput
+        control={control}
+        onMaxClick={() => setValue("amount", "1000000")}
+        {...args}
+      />
```

Comment:
why 2 `DollarInput`?

Reply:
So that you can see the focus change

Reply:
Well, if you click on the field and then oustide the field it also shows the focus in and focus out, so I don't think its necessary